smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolServiceClient.kt

```diff
@@ -145,26 +149,26 @@ open class HttpProtocolServiceClient(
     }
 
     private fun renderLogHandlerFactory(serviceSymbol: Symbol) {
-        writer.openBlock(
-            "public struct \$LLogHandlerFactory: \$N {",
-            "}",
-            serviceSymbol.name,
-            ClientRuntimeTypes.Core.SDKLogHandlerFactory
-        ) {
-            writer.write("public var label = \$S", serviceSymbol.name)
-            writer.write("let logLevel: \$N", ClientRuntimeTypes.Core.SDKLogLevel)
-
-            writer.openBlock("public func construct(label: String) -> LogHandler {", "}") {
-                writer.write("var handler = StreamLogHandler.standardOutput(label: label)")
-                writer.write("handler.logLevel = logLevel.toLoggerType()")
-                writer.write("return handler")
-            }
-
-            writer.openBlock("public init(logLevel: \$N) {", "}", ClientRuntimeTypes.Core.SDKLogLevel) {
-                writer.write("self.logLevel = logLevel")
-            }
-        }
-        writer.write("")
+//        writer.openBlock(
+//            "public struct \$LLogHandlerFactory: \$N {",
+//            "}",
+//            serviceSymbol.name,
+//            ClientRuntimeTypes.Core.SDKLogHandlerFactory
+//        ) {
+//            writer.write("public var label = \$S", serviceSymbol.name)
+//            writer.write("let logLevel: \$N", ClientRuntimeTypes.Core.SDKLogLevel)
+//
+//            writer.openBlock("public func construct(label: String) -> LogHandler {", "}") {
+//                writer.write("var handler = StreamLogHandler.standardOutput(label: label)")
+//                writer.write("handler.logLevel = logLevel.toLoggerType()")
+//                writer.write("return handler")
+//            }
+//
+//            writer.openBlock("public init(logLevel: \$N) {", "}", ClientRuntimeTypes.Core.SDKLogLevel) {
+//                writer.write("self.logLevel = logLevel")
+//            }
+//        }
+//        writer.write("")
```

Comment:
`LogHandlerFactoy` appears to be unused.  If that's the case, I will remove this.